cmd/release/utils/values/searchprs.go

```diff
@@ -0,0 +1,85 @@
+package values
+
+import (
+	"fmt"
+	"strings"
+	"context"
+
+	"github.com/google/go-github/v47/github"
+)
+
+func GetChangelogPRs(releaseVersion string) (string, error) {
+	githubClient := github.NewClient(nil)
+
+	ctx := context.Background()
+	opts := &github.SearchOptions{}
+	prs, _, err := githubClient.Search.Issues(ctx, "is:pr label:documentation repo:aws/eks-distro label:" + releaseVersion, opts)
+	if err != nil {
+		return "", fmt.Errorf("Getting PRs from %v: %v", githubClient, err)
+	}
+
+	lastDocRelease := prs.Issues[0].ClosedAt
+
+	patchPRs, _, err := githubClient.Search.Issues(ctx, "repo:aws/eks-distro is:pr is:merged label:patch label:" + releaseVersion + " merged:>" + fmt.Sprintf("%v", lastDocRelease), opts)
+
+	baseImgPRs, _, err := githubClient.Search.Issues(ctx, "repo:aws/eks-distro is:pr is:merged label:base-image-pkg-update label:" + releaseVersion + " merged:>" + fmt.Sprintf("%v", lastDocRelease), opts)
+
+	versPRs, _, err := githubClient.Search.Issues(ctx, "repo:aws/eks-distro is:pr is:merged label:dependencies label:" + releaseVersion + " merged:>" + fmt.Sprintf("%v", lastDocRelease), opts)
```

Comment:
Can we refactor these to be func that produced the string? It looks like the only difference between them is one of the labels. Also, we can fmt.Sprintf the whole string

Reply:
Could be refactored. Was considering also moving  the majority of the string to a constant file.